auth/signaturevalidator.js

```diff
@@ -101,7 +101,9 @@ var limitClientWithExt = function(credentialName, issuingClientId, accessToken,
     // Validate certificate scopes are subset of client
     if (!utils.scopeMatch(scopes, [cert.scopes])) {
       throw new Error("ext.certificate issuer `" + issuingClientId +
-                      "` doesn't have sufficient scopes");
+                      "` doesn't satisfiy all certificate scopes " +
+                      cert.scopes.join(', ') + ".  The temporary " +
+                      "credentials were not generated correctly.");
```

Comment:
If you look at the error in the bug, we're doing a lot of encoding of the json payload in the `message` attribute of another payload.  Not for fixing here, but .. yeah, we could use some cleanup.